experimental/PropertyDDS/packages/property-shared-tree-interop/src/schemaConverter.ts

```diff
@@ -11,7 +11,7 @@ import {
 	SchemaBuilder,
 	FieldKind,
 	Any,
-	TreeNodeSchema,
+	FlexTreeNodeSchema as TreeNodeSchema,
```

Comment:
Intended? Or VSCode trying to be helpful?

Reply:
Intended to keep the diff smaller.